code/game/objects/structures/crates_lockers/crates/large.dm

```diff
@@ -28,7 +28,8 @@
 	if(W.tool_behaviour == TOOL_CROWBAR)
 		if(manifest)
 			tear_manifest(user)
-
+		if(SEND_SIGNAL(src, COMSIG_CLOSET_PRE_OPEN, user, force) & BLOCK_OPEN)
```

Comment:
I don't think we should be using these signals in several places like this, maybe we can make use of the ``open()`` proc that calls this signal, and ``can_open()`` that it checks there?

Reply:
Upon further reflection, that was the actual best solution instead of me slamming in some signals. Whoops! Oh, well.